mmv1/products/vertexai/Schedule.yaml

```diff
@@ -0,0 +1,323 @@
+  - !ruby/object:Api::Type::NestedObject
+    name: timeSpecification
+    description: Required. The time specification to launch scheduled runs.
+    exactly_one_of:
+      - time_specification.0.cron
+    properties:
+      - !ruby/object:Api::Type::String
+        name: cron
+        description: |
+          The scheduled run time based on the user-specified schedule.
+          A timestamp in RFC3339 UTC "Zulu" format, with nanosecond
+          resolution and up to nine fractional digits. Examples:
+          "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+  - !ruby/object:Api::Type::NestedObject
+    name: request
+    description: |
+      Required. The API request template to launch the scheduled runs. 
+      User-specified ID is not supported in the request template.
+    exactly_one_of:
+      - request.0.create_pipeline_job_request
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: createPipelineJobRequest
+        description: |
+          Request for PipelineService.CreatePipelineJob.
+          CreatePipelineJobRequest.parent field is required
+          (format: projects/{project}/locations/{location}).
+        properties:
+          - !ruby/object:Api::Type::String
+            name: parent
+            description: Required. The resource name of the Location to create the PipelineJob in.
+              Format: projects/{project}/locations/{location}
+            required: true
+          - !ruby/object:Api::Type::NestedObject
+            name: pipelineJob
+            description: Required. The PipelineJob to create.
+            required: true
+            properties:
+              - !ruby/object:Api::Type::String
+                name: displayName
+                description: |
+                  The display name of the Pipeline. The name can be up to 128 characters long
+                  and can consist of any UTF-8 characters.
+              - !ruby/object:Api::Type::NestedObject
+                name: pipelineSpec
+                description: The spec of the pipeline.
+                required: true
+                # TODO: add pipelineSpec properties
+              - !ruby/object:Api::Type::KeyValuePairs
+                name: labels
+                description: |
+                  The labels with user-defined metadata to organize PipelineJob.
+                  Label keys and values can be no longer than 64 characters
+                  (Unicode codepoints), can only contain lowercase letters,
+                  numeric characters, underscores and dashes. International
+                  characters are allowed. See https://goo.gl/xmQnxf for more
+                  information and examples of labels.
+              - !ruby/object:Api::Type::NestedObject
+                name: runtimeConfig
+                description: Runtime config of the pipeline.
+                required: true
+                properties:
+                  # TODO parameters
```

Comment:
Not sure how to implement this type in magic modules
map (key: string, value: object (Value))

https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/projects.locations.pipelineJobs#PipelineJob.RuntimeConfig